### PR TITLE
6280 - Datagrid page size selector pptr test

### LINKED ISSUE #6280: Unit Testing: Create a Puppeteer Script for "DataGrid: 'hideOnOnePage' should keep the page size selector for non-empty state"
Describe the Issue
Create a unit testing for "DataGrid: 'hideOnOnePage' should keep the page size selector for non-empty state".

See attached BDD for more information.
[BDD_github#3706.docx](https://github.com/infor-design/enterprise/files/8426527/BDD_github.3706.docx)

Related github issue
#3706

### PR DESCRIPTION
**Pull request by jbrcna**
**Explain the _details_ for making this change. What existing problem does the pull request solve?**

**Related github/jira issue (required)**:
closes #6280 
related to #3706 

**Steps necessary to review your pull request (required)**:

1. pull this branch
2. build and run the app
3. run npm run e2e:puppeteer Datagrid
4. It should pass.

Steps 
Given user is on https://main-enterprise.demo.design.infor.com/components/datagrid/test-count-in-select-all-current-page-setting.html
When Select page size to 100
Then Page size selector should still be visible
And the pager should be hidden

**Included in this Pull Request**:
- [ ] An e2e or functional test for the bug or feature.
- [ ] A note to the change log.

### CODE REVIEW

File: test/components/datagrid/datagrid.puppeteer-spec.js [hunk 1]
```diff
@@ -114,3 +114,37 @@ describe('Datagrid test to keep the column strech in responsive view', () => {
     expect(img).toMatchImageSnapshot(config);
   });
 });
+
+describe('Datagrid test-count-in-select-all-current-page-setting tests', () => {
```
Comment (tmcconechy) [inline test/components/datagrid/datagrid.puppeteer-spec.js:118]: Minor but maybe a better name?

Comment (jbrcna) [reply test/components/datagrid/datagrid.puppeteer-spec.js:118]: hello @tmcconechy. I was thinking of "Datagrid test to  hide pager if records per page higher than total items" or maybe "Datagrid hide on One Page tests"

Comment (tmcconechy) [reply test/components/datagrid/datagrid.puppeteer-spec.js:118]: I think keep it short maybe "Datagrid test cont and select all" something like that

Comment (jbrcna) [reply test/components/datagrid/datagrid.puppeteer-spec.js:118]: oh okay you mean "Datagrid test-count-in-select-all tests"? just making it short and not changed the whole title.

Comment (tmcconechy) [reply test/components/datagrid/datagrid.puppeteer-spec.js:118]: I think we should keep the test descriptions in english mostly so no dashes

Comment (jbrcna) [reply test/components/datagrid/datagrid.puppeteer-spec.js:118]: ok got it. will do it now, making it as "Datagrid test count in select all tests". thank you :smiley:

**Review by tjamesallen15 — APPROVED** [review]

**Pull request merged**